kaznet/apps/main/tests/test_api.py

```diff
@@ -70,7 +70,7 @@ def _create_instance(self, mocked, pending: bool = False):
             ],
             "_status": "submitted_via_web",
             "_review_status": settings.ONA_SUBMISSION_REVIEW_APPROVED,
-            "_review_comments": "This is a review comment",
+            "_review_comment": "This is a review comment",
```

Comment:
Why is this changing?

Reply:
`_review_comments` in stead of `_review_comment` is a typo because `onadata` uses `_review_comment`